packages/framework/src/Form/Admin/Country/CountryFormType.php

```diff
@@ -106,6 +110,61 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'required' => false,
                 'label' => t('Priority'),
             ])
+
+            // @todo: only for testing, remove before merge
```

Comment:
You should fix it?

Reply:
I need this for testing, after that, I will remove this code.